### ISSUE TITLE
Minecraft: Kids are not playing together online saying they need "realms"

### ISSUE THREAD

**Issue by 130s** [kinu-garage/hut_10sqft#1051]
After https://github.com/kinu-garage/hut_10sqft/issues/1042 they no longer see error but they are not yet playing on the same "world" on MC. They showed me [this](https://photos.google.com/album/AF1QipNLiSS2hdM-FBvgNBF2ZcdxwYa6b4U529S-6B6P/photo/AF1QipNwFOjK2j_bBtJFjuvg60wh0HVjuVwiQORHkWUd), which shows $7.99/mo.  
![image](https://github.com/kinu-garage/hut_10sqft/assets/1840401/3f771106-c9da-4f01-9c6f-7847bb13de20)

**Comment by 130s** [comment]
TLDR; I have a feeling that to enable multiple players playing Minecraft together requires the server software to be accessible from each player's MC client, and that s/w is called `Realms`. With the [paid subscription (minecraft.net)](https://www.minecraft.net/en-us/realms) we can have a private (i.e. no one else is accessing than invited users) running server on cloud, and there's an option to run it on my own (which is likely free?).

## Getting understanding

[How to play multiplayer in Minecraft, using either a public server or one you create yourself (businessinsider.com)](https://www.businessinsider.com/guides/tech/how-to-play-multiplayer-in-minecraft-java#:~:text=There%20are%20three%20ways%20to,Minecraft%20server%20for%20local%20multiplayer.) says:
> How you play multiplayer Minecraft will vary depending on whether you have the newer, console-friendly Bedrock edition, or the classic Java edition. Whichever edition you have, you can only play with friends who have the same version.
> If that version is Java, there are few different ways to go about playing with your friends.
> Whether you feel up to the task of joining a public server or creating your own server (which is less scary than it sounds), want to host a local LAN server, or would prefer to use the paid "Realms" server-hosting option, hosting a multiplayer Minecraft game should only take a short time.

The rest of this article shows things to do to run my own server using Java edition, which I assume Bedrock edition client can connect.

## Running my own server on RPi
Interested in setting up a globally accessible server, using Raspberry Pi. [tomshardware.com recommends 2GB RAM minimum on RPi 3 or even 4](https://www.tomshardware.com/how-to/raspberry-pi-minecraft-server) but someone [says 512 Mb is even fine](https://forums.raspberrypi.com/viewtopic.php?t=317931).

## Public servers
May be another option.
List of pub servers https://minecraft-mp.com/type/realms/ I'm assuming joining these are free.

**Comment by 130s** [comment]
In a kind of random manner one of my kids chose this server and started playing. `Play -> Server tab` -> URL and port.

Address: best.fadecloud.com
Port: 19132
